test/test_ride.py

```diff
@@ -0,0 +1,185 @@
+import unittest
+from app.models.rides import rides
+
+
+obj = rides()
+class rideTest(unittest.TestCase):
+	def setUp(self):
+		self.ride_data = {
+		'username':'kwame',
+		'car capacity':3,
+		'contact':'072345678',
+		'time':'12:30pm',
+		'date':12,
+		'month':'june',
+		'year':2018,
+		'destination': 'kasarani',
+		'pickup':'mwiki'
+		}
+
+		self.request_data = {
+		'name':'kwame',
+		'pickup time': '12:30pm',
+		'destination': 'kasarani',
+		'pick up point': 'mwiki'
+		}
+
+
+
+	#test adding a ride
+	def test_adding_ride(self):
+		result = self.ride_data
+		self.assertTrue(obj.add_ride(
+			result['username'],
+			result['car capacity'],
+			result['contact'],
+			result['time'],
+			result['date'],
+			result['month'],
+			result['year'],
+			result['destination'],
+			result['pickup']
+			))
+	#Happy test
+	def test_username(self):
+		self.assertIn('username',self.ride_data)
```

Comment:
why are you asserting the keys in your dictionary????

Reply:
Am testing the key so that i can limit the users from posting the wrong json object(eg {'house': 'someones name') i.e i want all the json object to contain username keys that will store the user name as its value, pick up point that will store the pick up point for the ride as its value etc